tasks/system.yml

```diff
@@ -76,10 +89,10 @@
 # Set Timezone to US/Pacific (default was: 'Etc/UTC')
 # XXX: Only a single test requires this in plone.app.form 2.2.x
 # https://github.com/plone/plone.app.form/blob/2.2.x/plone/app/form/widgets/datecomponents.txt
```

Comment:
seems that this is no longer true on `master` branch. We could remove the comment then 👍🏾